packages/cli/src/networks/index.ts

```diff
@@ -121,8 +127,13 @@ export async function getNetworkBootnodes(network: NetworkName): Promise<string[
       const bootEnrs = await fetchBootnodes(network);
       bootnodes.push(...bootEnrs);
     } catch (e) {
-      // eslint-disable-next-line no-console
-      console.error(`Error fetching latest bootnodes: ${(e as Error).stack}`);
+      if (e instanceof FetchBootFileError) {
```

Comment:
If I added the logic, the nested Error mesage resolved.

```suggestion
     if (e instanceof FetchBootFileError) {
        // eslint-disable-next-line no-console
        console.error(`${(e as Error).stack}`);
      }
       if (e instanceof FetchError) {
        console.error(`${(e as Error).stack}`);
       }
```
The message becomes 

```
FetchError: Request to https://example-example-example.com/notfound failed, reason: getaddrinfo ENOTFOUND example-example-example.com
    at wrappedFetch (file:///home/hnaito/lodestar/packages/api/src/utils/client/fetch.ts:12:11)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at fetchBootnodes (file:///home/hnaito/lodestar/packages/cli/src/networks/index.ts:114:15)
    at getNetworkBootnodes (file:///home/hnaito/lodestar/packages/cli/src/networks/index.ts:128:24)
    at beaconHandlerInit (file:///home/hnaito/lodestar/packages/cli/src/cmds/beacon/handler.ts:179:35)
    at Object.beaconHandler [as handler] (file:///home/hnaito/lodestar/packages/cli/src/cmds/beacon/handler.ts:36:84)
```